<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardDialog</class>
 <widget class="QDialog" name="BoardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>493</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,3,1,3,10,3">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rows:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_box_row">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Columns:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_box_col">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Game Mode:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_box_mode"/>
    </item>
    <item>
     <widget class="QPushButton" name="generate_button">
      <property name="text">
       <string>Generate Board</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>580</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="genValueButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="valueLabel">
      <property name="text">
       <string>##</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="passButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Pass</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset Board</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>620</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Current Player: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="currentPlayerLabel">
      <property name="styleSheet">
       <string notr="true">color:#ff5555</string>
      </property>
      <property name="text">
       <string>Player 1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
